### DISCUSSION TITLE
How to change the css-logic from 4. to 5.

### DISCUSSION THREAD

**Discussion by phillippppppo** [ant-design/ant-design#46240] [Q&A]
Hi,

I got my antd.theme.less and my antd.theme.css. The css is imported into my App.tsx and everything worked well. Now the logic for the css had changed, when changing to 5. How do I migrate my old css logic into 5., because for now it doesn´t work no more and no css is applied to my application.

Thank you!

**Comment by zombieJ** [comment]
You can check this to generate static less variables: https://ant.design/docs/react/customize-theme#static-consume-eg-less
